backends/files.go

```diff
@@ -59,6 +61,18 @@ func NewFiles(authOpts map[string]string, logLevel log.Level) (Files, error) {
 		return files, errors.New("Files backend error: no password path given")
 	}
 
+	if saltEncoding, ok := authOpts["salt_encoding"]; ok {
+		if saltEncoding == "base64"  {
+			files.SaltEncoding = saltEncoding
+			log.Infof("files backend: set salt encoding to: %s", saltEncoding)
+		} else if saltEncoding == "utf-8" {
+			files.SaltEncoding = saltEncoding
+			log.Infof("files backend: set salt encoding to: %s", saltEncoding)
+		} else {
+			log.Errorf("files backend: invalid salt encoding specified: %s", saltEncoding)
+		}
+	}
```

Comment:
Since they are already defined in the `common` package, we could just use the constants to switch over the encoding, e.g.:
```
switch saltEncoding{
		case common.Base64, common.UTF8:
			files.SaltEncoding = saltEncoding
			log.Debugf("files backend: set salt encoding to: %s", saltEncoding)
		default:
			log.Errorf("files backend: invalid salt encoding specified: %s, will default to base64 instead", saltEncoding)
}
```
Notice also that I changed the logging to debug level. This should be used in every backend.

Reply:
Thanks, this is much cleaner, will add to other backends.

Am I right in thinking that debug logging will show no output by default? I'd set this to Info to ensure user got visual feedback that salt encoding was configured correctly.

Reply:
It won't show unless you explicitly set the log level to debug as part of the the options.